docs/pages/setup/helm-reference/teleport-cluster.mdx

```diff
@@ -63,51 +63,82 @@ This reference details available values for the `teleport-cluster` chart.
 
 `kubeClusterName` sets the name used for the Kubernetes cluster. This name will be shown to Teleport users connecting to the cluster.
 
-## `authenticationType`
+## `authentication`
+
+### `authentication.type`
 
 | Type | Default value | Required? | `teleport.yaml` equivalent | Can be used in `custom` mode? |
 | - | - | - | - | - |
 | `string` | `local` | Yes | `auth_service.authentication.type` | ❌ |
 
-`authenticationType` controls the authentication scheme used by Teleport. Possible values are `local` and `github` for OSS, plus `oidc`, `saml`, and `false` for Enterprise.
+`authentication.type` controls the authentication scheme used by Teleport. Possible values are `local` and `github` for OSS, plus `oidc` and `saml` for Enterprise.
+
+### `authentication.connectorName`
+
+| Type | Default value | Required? | `teleport.yaml` equivalent | Can be used in `custom` mode? |
+| - | - | - | - | - |
+| `string` | `""` | No | `auth_service.authentication.connector_name` | ❌ |
+
+`authentication.connectorName` sets the default authentication connector.
+See [the SSO documentation](../../enterprise/sso.mdx). Other possible values are `local` and
+[`passwordless`](../../access-controls/guides/passwordless.mdx#optional-enable-passwordless-by-default).
```

Comment:
Should the other values have links as well?

Reply:
Done.